library/x509write_csr.c

```diff
@@ -204,28 +218,49 @@ int mbedtls_x509write_csr_der( mbedtls_x509write_csr *ctx, unsigned char *buf, s
         return( MBEDTLS_ERR_X509_INVALID_ALG );
 
     if( ( ret = mbedtls_oid_get_oid_by_sig_alg( pk_alg, ctx->md_alg,
-                                                &sig_oid, &sig_oid_len ) ) != 0 )
+                                              &sig_oid, &sig_oid_len ) ) != 0 )
```

Comment:
The parameter alignment here doesn't seem right.
The previous alignment is as expected

Reply:
The previous alignment is correct even if it exceeds 80 characters?